src/playground/components/Configuration.js

```diff
@@ -184,10 +183,12 @@ export default function Configuration({ rulesMeta, eslintVersion, onUpdate, opti
                                 isSearchable={false}
                                 styles={customStyles}
                                 theme={theme => customTheme(theme)}
-                                defaultValue={ECMAVersionsOptions.filter(ecmaVersion => options.parserOptions.ecmaVersion === ecmaVersion.value)}
+                                defaultValue={
+                                    ECMAVersionsOptions.filter(ecmaVersion => ecmaVersion.value === (options.languageOptions.ecmaVersion || "latest"))
```

Comment:
Set to "latest" if it's invalid?
```suggestion
                                    ECMAVersionsOptions.filter(ecmaVersion => ecmaVersion.value === (options.languageOptions.ecmaVersion)) || "latest"
```